src/shogun/regression/LeastAngleRegression.h

```diff
@@ -120,14 +126,16 @@ class CLeastAngleRegression: public CLinearMachine
 	 *
 	 * @param num_variable number of non-zero coefficients
 	 */
+	
 	void switch_w(int32_t num_variable)
 	{
 		if (w.vlen <= 0)
-			SG_ERROR("cannot swith estimator before training")
+			SG_ERROR("cannot switch estimator before training")
```

Comment:
"Train model before updating model parameters"  .. something like this sounds good ?